src/tasks/mixin/map_mixin.py

```
@@ -136,71 +118,42 @@ def clear_icon_in_map(self, need_reserve_icon_name=None, ocr=False):
 
         return True
 
-    def to_near_transfer_point(self, test_target_box):
+    def to_near_transfer_point(self):
         """
         在地图上寻找最近的传送点并执行传送。
 
         流程：
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)